install/fossinit.php

```diff
@@ -449,10 +474,11 @@ function initLicenseRefTable($Verbose)
       $rf_flag_check = $row_check['rf_flag'];
 
       $sql = "UPDATE license_ref set ";
-      if($rf_flag_check == 1 ||  $rf_flag == 1) {
-        if ($rf_text_check != $rf_text && !empty($rf_text) && !(stristr($rf_text, 'License by Nomos')))  $sql .= " rf_text='$rf_text', rf_flag='1',";
-      } else {
+      if( ($rf_flag_check == 1 &&  $rf_flag == 1) ||
```

Comment:
If the `$rf_flag_check = 1` and `$rf_flag = 1`, then the license is text is not updated manually by the user. So in this case the `rf_text` should be updated.

I think there should be **only one** condition in the `if` to check `if($rf_flag_check == 2 &&  $rf_flag == 1)` (license text was updated by UI).